### DISCUSSION TITLE
有两个选项没太弄明白

### DISCUSSION THREAD

**Discussion by NightSharp** [the1812/Bilibili-Evolved#5063] [提问 / Q&A]
请问这两个选项的不同设置会对体验造成哪些影响，或者说，具体会影响到哪些方面
![图片](https://github.com/user-attachments/assets/ea7edb14-c35a-4727-8737-ef28f2226649)

**Comment by the1812** [comment]
`同时`: 在 [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event) 时执行, 会和 b 站自身的初始化代码一起执行
`延后`: 在 [load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) 时执行, 等 b 站代码运行完再执行

在性能较差的设备上, 使用 `同时` 会造成比较长的白屏 / 骨架屏时间, 换成 `延后` 可以先看到 b 站原版的界面出来, 然后脚本的功能再出现. 不过以现在的主流性能来说开哪个都没啥区别了 ~~除非你用的是 Intel UHD Graphics~~

**Reply by NightSharp** [reply]
> `同时`: 在 [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event) 时执行, 会和 b 站自身的初始化代码一起执行 `延后`: 在 [load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) 时执行, 等 b 站代码运行完再执行
> 
> 在性能较差的设备上, 使用 `同时` 会造成比较长的白屏 / 骨架屏时间, 换成 `延后` 可以先看到 b 站原版的界面出来, 然后脚本的功能再出现. 不过以现在的主流性能来说开哪个都没啥区别了 ~除非你用的是 Intel UHD Graphics~

感谢解答。
我现在是功能同时样式延后，不知道为啥反过来时会卡一些
